src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
I guess the interesting diff is at https://github.com/pgilad/jquery/commit/ab692d396542b1a65a93460359fe1d8c6ca6ccf6.